@xen-orchestra/web-core/lib/components/card/CardTitle.vue

```diff
@@ -0,0 +1,64 @@
+<!-- v1.0 -->
+<template>
+  <div class="card-title">
+    <div class="left-blok">
+      <div class="title-and-counter">
+        <span class="typo h6-medium"><slot name="left"></slot></span>
+        <UiCounter v-if="count" class="count" :value="count" :color="counterColor" />
+      </div>
+      <span class="typo p3-regular bottom-left"><slot name="bottom-left"></slot></span>
```

Comment:
Don't forget to add a `v-if` check to avoid adding the `<span>` if the slot content is not provided (I also renamed the slot and class names to match my comment on the file):
```suggestion
      <span v-if="$slots.description" class="typo p3-regular description"><slot name="description" /></span>
```